.openshift-ci/build/build-operator-artifacts.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+# Execute all build steps required to create the operator bundle.tar.gz and
+# scripts used in image/rhel/Dockerfile
+
+ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")"/../.. && pwd)"
+source "$ROOT/scripts/ci/lib.sh"
+
+set -euo pipefail
+
+build_operator_bundle_and_binary() {
+    # avoid a -dirty tag
+    info "Reset to remove Dockerfile modification by OpenShift CI"
```

Comment:
How does the CI adjust the dockerfiles? 🤔 
This seems odd to me tbh

Reply:
ci-operator has this mechanism of injecting images into the `FROM ...` directives. Seems to be common practice in OSCI world, we also use it in asc-fleet-manager. But, it is also news to me that this works by patching the unpacked source code and leaving it like that.

Reply:
CI updates the `FROM`(s)